oauth2client/_helpers.py

```diff
@@ -39,15 +39,42 @@ def _json_encode(data):
   return json.dumps(data, separators=(',', ':'))
 
 
+def _to_bytes(value, encoding='ascii'):
```

Comment:
I'm inherently skeptical of optional arguments in private functions, and I think the call sites of this function are more clear if the encoding is present and explicit at each one.